src/components/List.js

```diff
@@ -1,7 +1,15 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
+import { updatePurchaseDate } from '../lib/firebase.js';
 
-const List = ({ token, results }) => {
+const List = ({ results, token }) => {
+  function handleOnCheck(event) {
+    updatePurchaseDate(token, event.target.value);
+  }
+
+  function checkTime(time) {
+    return Date.now() - time <= 86400000; //number of milliseconds equal to 24 hours
+  }
```

Comment:
This takes the UTC time the purchased item was checked off the list and subtracts it from the current UTC time. If the difference is greater than or equal to 1 day (86400000), the function will return true.